src/Storages/MergeTree/MergeTreeData.cpp

```diff
@@ -1244,16 +1244,6 @@ MergeTreeData::LoadPartResult MergeTreeData::loadDataPart(
             .withPartFormatFromDisk()
             .build();
     }
-    catch (const Exception & e)
-    {
-        /// Don't count the part as broken if there was a retryalbe error
-        /// during loading, such as "not enough memory" or network error.
-        if (isRetryableException(e))
-            throw;
-
-        mark_broken();
-        return res;
-    }
     catch (const Poco::Net::NetException &)
```

Comment:
Should we add `Poco::Net::NetException` and `Poco::TimeoutException` to `isRetryableException` too?

Reply:
addressed it